change/@fluentui-react-portal-db1a41bd-bcbd-4135-a22e-9a04232064c4.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "none",
```

It's not affecting published artifacts in any way, so "none" is more reasonable